NF.csv

```diff
@@ -601,6 +600,17 @@ Ganglioglioma,GNG,,,FALSE,ONCOTREE,tumorType,,,,,,,,,,
 Not Applicable,,,,FALSE,Sage Bionetworks,tumorType,,,,,,,,,,
+Colorectal Carcinoma,A malignant epithelial neoplasm that arrises from the colon or rectum and invates through the muscularis mucosa into the submucosa.,,,FALSE,obo:NCIT_C2955,tumorType,,,,,,,,,,
+Recurrent MPNST,The reemergence of malignant peripheral nerve sheath tumor after a period of remission.,,,FALSE,obo:NCIT_C8823,tumorType,,,,,,,,,, 
+Juvenile Myelomonocytic Leukemia,A myelodysplastic/myeloproliferative neoplasm of childhood that is characterized by proliferation principally of the granulocytic and monocytic lineages,,,FALSE,obo:NCIT_C9233,tumorType,,,,,,,,,,
+Sarcoma,"An usually aggressive malignant neoplasm of the soft tissue or bone...Representative subtypes are liposarcoma, leiomyosarcoma, osteosarcoma, and chondrosarcoma.",,,FALSE,obo:NCIT_C9118,tumorType,,,,,,,,,,
+Fibromatosis,A poorly circumscribed neoplasm arising from the soft tissues. It is characterized by the presence of spindle-shaped fibroblasts and an infiltrative growth pattern.,,,FALSE,obo:NCIT_C3042,tumorType,,,,,,,,,,
+Fibrosarcoma,A malignant mesenchymal fibroblastic neoplasm affecting the soft tissue and bone.,,,FALSE,obo:NCIT_C3043,tumorType,,,,,,,,,,
+Necrotic Neoplasm,A neoplasm characterized by focal or diffuse tumor cell necrosis.,,,FALSE,obo:NCIT_C36029,tumorType,,,,,,,,,,
+Hemorrhagic Neoplasm,A neoplasm with hemorrhagic findings.,,,FALSE,Sage Bionetworks,tumorType,,,,,,,,,,This is based after the NCIT term 'Necrotic Neoplasm' to accomodate records using only 'hemorrhage' for tumor description -- it may be of limited usefulness long-term.
+NF1-Associated Tumor,"Unspecified NF1-associated tumor; use a more specific tumor classification if available.",,,FALSE,Sage Bionetworks,tumorType,,,,,,,,,,This was added to accomodate records that only specified 'NF1' and is patterned after the existence of the NCIT term 'NF1-Associated Malignant Peripheral Nerve Sheath Tumor'
+NF2-Associated Tumor,"Unspecified NF2-associated tumor; use a more specific tumor classification if available.",,,FALSE,Sage Bionetworks,tumorType,,,,,,,,,,This was added to accomodate records that only specified 'NF2' and is patterned after the existence of the NCIT term 'NF1-Associated Malignant Peripheral Nerve Sheath Tumor'
+ANNUBP,"Atypical neurofibromatous neoplasms of uncertain biologic potential is a provisional classification for lesions displaying at least two features: atypia, loss of neurofibroma architecture, high cellularity, and/or mitotic activity >1/50 but <3/10 high power fields. They can be inconsistently diagnosed as atypical neurofibroma or low-grade MPNST. See https://pubmed.ncbi.nlm.nih.gov/28551330/.",,,FALSE,Sage Bionetworks,tumorType,,,,,,,,,,
```

Comment:
We should definitely include ANNUBP. I am wondering what the semantic overlap between ANNUBP and Atypical Neurofibroma is. Can we treat these as the same? IIRC the Atypical Neurofibroma term comes from JHU. If possible, I'd like to merge these two terms.

Reply:
In the paper the authors want to distinguish it from Atypical Neurofibroma, which is why the term is added. Sounds it should be a subclass.